packs/salt/actions/local.py

```diff
@@ -14,7 +14,10 @@ def run(self, **kwargs):
             st2 run salt.client module=pkg.install \
                     kwargs='{"pkgs":["git","httpd"]}'
         '''
-        self.generate_package(cmd=cmd)
+        # TODO: This is broken, fix it. I temporary disabled it to avoid pylint
```

Comment:
@Kami `kwargs` will be used for the explicit actions that will be defined in #186. I made edits to this exact line. I'll rebase, resolve and address the non-use of kwargs.